.github/workflows/docker-build-push-google.yaml

```diff
@@ -43,6 +43,11 @@ on:
         required: false
         type: string
         default: .
+      file:
```

Comment:
@rhudev  you have to use that parameter like context. Check

Reply:
consider moviong another Dockerfile to `<my_scope>/Dockerfile`

Reply:
as discussed this does not work, file will be needed @jazzlyn

Reply:
its possible to set file with path. Check: https://github.com/strg-at/methmedia-nuxt-blickpunktjuwelier/pull/6/files#diff-2da8ac5d7474f581aef3c6d2dfdc4764592f50f418e2b108fd383d7747cbea3cR32